source/core/TDCodecVia.cpp

```diff
@@ -2329,6 +2329,56 @@ void TDViaFormatter::FormatClusterData(TypeRef type, void *pData)
     _string->Append(Fmt()._clusterPost);
 }
 //------------------------------------------------------------
+void TDViaFormatter::FormatVariant(TypeRef type, void *pData)
```

Comment:
I would add a test to print for nested variant with attributes. You may just add print to one of the recently added nested variant test via files.

Reply:
Done. Added new test in `VariantWithVariantAsAttribute`

Reply:
Done.